crates/subspace-core-primitives/src/lib.rs

```diff
@@ -461,6 +461,8 @@ pub type PieceIndex = u64;
 
 /// Hash of `PieceIndex`
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
```

Comment:
I generally prefer this to be more specific:
```suggestion
#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
```

BTW why do you need it? Also `camelCase` is unnecessary here as it does nothing.

Reply:
Fixed here: [74266e9](https://github.com/subspace/subspace/pull/541/commits/74266e94b9393b1ce550d1f0b9439233c1d24895)